docs/getting-started-auditor.md

```diff
@@ -0,0 +1,170 @@
+# Getting Started with Scalar DL Auditor
+
+This document explains how to get started with Scalar DL Auditor.
+Here, we assume that you have already finished reading the following guides and gained some experiences in overall Scalar DL.
```

Comment:
Just minor grammar point. Normally it is  "experience" in singular form.  https://textranch.com/103737/gained-experiences/or/gained-experience/

Reply:
Thank you! Fixed in 9c312f5aab1518f1c2a84800bdd1ea1287d142d0.